board/miyoo/boot/firstboot

```diff
@@ -610,6 +610,7 @@ if $CHANGE_FS_P4 && test "${P4_FS_TYPE}" == "EXT4" ; then
 	sleep 1
 	mount -o remount,rw / /
 	sed -i "s|\(::sysinit:/bin/mount /dev/mmcblk0p4\).*|\1 "${HOME}" -o rw,sync,exec|" /etc/inittab
+	sed -i 's|^::sysinit:/sbin/modprobe btrfs|#&|' /etc/inittab
```

Comment:
rebase off-branch, firstboot had severe change

also mmcblk0p5 can be BTRFS so you have to sed it there also and only if not P4_FS_TYPE=BTRFS or P5_FS_TYPE=BTRFS